backend/models/refund_request.py

```
@@ -0,0 +1,1022 @@
+            "created_at": {"$lt": cutoff_time}
+        }).to_list(length=None)
+        
+        completed_count = 0
+        rolled_back_count = 0
+        failed_count = 0
+        
+        for refund in stale_refunds:
+            try:
+                transaction_id = refund.get("transaction_id")
+                request_id = refund.get("request_id")
+                amount = refund.get("payment_amount", 0)
```

Ruff (0.14.2)

Local variable `amount` is assigned to but never used

Remove assignment to unused variable `amount`

(F841)